script/release/DSL-scripts/01.KIE_pushReleaseBranches.sh

```diff
@@ -0,0 +1,81 @@
+# clone the build-bootstrap that contains the other build scripts
+if [ "$SOURCE" == "community-branch" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
+
+   # clone rest of the repos
+   ./droolsjbpm-build-bootstrap/script/git-clone-others.sh --branch $BASE_BRANCH --depth 70
+   
+   # checkout to local release names
+   ./droolsjbpm-build-bootstrap/script/git-all.sh checkout -b $RELEASE_BRANCH $BASE_BRANCH
+   
+   # add new remote pointing to jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-add-remote-jboss-integration.sh
+
+fi
+
+if [ "$SOURCE" == "community-tag" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
+
+   # clone rest of the repos
+   ./droolsjbpm-build-bootstrap/script/git-clone-others.sh --branch $BASE_BRANCH --depth 70
+   
+   # add new remote pointing to jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-add-remote-jboss-integration.sh
+   
+   # get the tags of community
+   ./droolsjbpm-build-bootstrap/script/git-all.sh fetch --tags origin
+   
+   # checkout to local release names
+   ./droolsjbpm-build-bootstrap/script/git-all.sh checkout -b $RELEASE_BRANCH $TAG
+
+fi
+   
+if [ "$SOURCE" == "production-tag" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from jboss-integration
+   git clone git@github.com:jboss-integration/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
+
+   # clone rest of the repos
+   ./droolsjbpm-build-bootstrap/script/git-clone-others.sh --branch $BASE_BRANCH --depth 70
+   
+   # add new remote pointing to jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-add-remote-jboss-integration.sh
+   
+   # get the tags of jboss-integration
+   ./droolsjbpm-build-bootstrap/script/git-all.sh fetch --tags jboss-integration
+   
+   # checkout to local release names
+   ./droolsjbpm-build-bootstrap/script/git-all.sh checkout -b $RELEASE_BRANCH $TAG
+
+fi
+
+# upgrades the version to the release/tag version
+./droolsjbpm-build-bootstrap/script/release/update-version-all.sh $RELEASE_VERSION $TARGET
+
+# update kie-parent-metadata
+cd droolsjbpm-build-bootstrap/
+
+# change <version.org.uberfire>, <version.org.dashbuilder> and <version.org.jboss.errai>
+sed -i "$!N;s/<version.org.uberfire>.*.<\/version.org.uberfire>/<version.org.uberfire>$UBERFIRE_VERSION<\/version.org.uberfire>/;P;D" pom.xml
+sed -i "$!N;s/<version.org.dashbuilder>.*.<\/version.org.dashbuilder>/<version.org.dashbuilder>$DASHBUILDER_VERSION<\/version.org.dashbuilder>/;P;D" pom.xml
+sed -i "$!N;s/<version.org.jboss.errai>.*.<\/version.org.jboss.errai>/<version.org.jboss.errai>$ERRAI_VERSION<\/version.org.jboss.errai>/;P;D" pom.xml
+
```

Comment:
this has to be done as <version.org.uberfire>, <version.org.dashbuilder> and <version.org.jboss.errai> in kie-parent pom are not modified automatically when upgrading the version - so it has to be done "manually". The update version is in and should be in the push release branches job.